src/components/CreateProposal/SchemeForms/CreateContributionRewardProposal.tsx

```diff
@@ -63,15 +67,17 @@ class CreateContributionReward extends React.Component<IProps, null> {
     if (!values.beneficiary.startsWith("0x")) { values.beneficiary = "0x" + values.beneficiary; }
 
     const proposalValues = {...values,
-      type: IProposalType.ContributionReward,
+      // type: IProposalType.ContributionReward,
```

Comment:
ok, apart from keeping useless commented lines in the code (bad habit of mine, sorry), the explanatino is this. Since we now have different versions of schemes deployed, we cannot create a  proposal by just specifiying it is a "contributionreward" proposal (for example). There could be many such schemes deployed. So instead, we must specify the address of the scheme we are creating a proposal in.